src/core.js

```diff
@@ -30,7 +30,7 @@ define([
     'jquery.jstree',
     'jquery.bootstrap',
     'jquery.fancybox',  // only thing we use fancybox for is its spinner, no actual display of anything
-    'jquery-ui',  // used for buttons in Edit Source XML, and dialogs
+    'jquery-ui',  // used for autocomplete
```

Comment:
This works with inputs, but unfortunately won't work with content editable divs.

Took a quick look and it seems like there are plugins that handle that, but I'm not sure how well those will work.